docker/test/libfuzzer/run_libfuzzer.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -eu
```

Comment:
I'd suggest moving either everything to bash or to python. It will be a nightmare to have a mixed environment. Like we have style check right now